cogito/putter_private_test.go

```diff
@@ -143,32 +146,34 @@ func TestCheckGitRepoDirFailure(t *testing.T) {
 		{
 			name:    "repo with unrelated HTTPS remote",
 			dir:     "testdata/one-repo/a-repo",
-			repoURL: testhelp.HttpsRemote("owner-a", "repo-a"),
+			repoURL: testhelp.HttpsRemote("github.com", "owner-a", "repo-a"),
 			wantErrWild: `the received git repository is incompatible with the Cogito configuration.
 
 Git repository configuration (received as 'inputs:' in this PUT step):
-      url: https://github.com/owner-a/repo-a.git
+    url: https://github.com/owner-a/repo-a.git
     owner: owner-a
-     repo: repo-a
+    repo: repo-a
```

Comment:
Do we know why these whitespace changes are needed? (here and everywhere else in this PR)
I have a recollection that there are tabs in the output, if this is correct, then maybe all these changes are not needed if we fix where we create the output?

Reply:
> Do we know why these whitespace changes are needed? (here and everywhere else in this PR)

they are not needed; I just wanted to make them consistent. You mentioned that these were tabs but they were not tabs (at least this is my understanding). In my editor I see; 4 spaces, 5 spaces, 6 spaces; so I changed it such that always its 4 spaces.

Reply:
OK.

Reply:
No actions were taken on this point. I double-checked again and for me, it seems that the changes are correct because they try to make the spaces consistent (4 spaces everywhere)